site-content/source/modules/ROOT/pages/events/20221222-cday-china.adoc

```diff
@@ -0,0 +1,41 @@
+= Cassandra Day China announced
+:page-layout: event-post
+:page-role: blog-post
+:page-post-date: December 6, 2022
+:page-post-author: Tom Lu
+:description: Cassandra Day China
+:keywords: cassandradays china
+
+image::events/20221222-cday-china-1024x512.png[Cassandra Day China]
+
+=== Calling all developers!
+
+The Apache Cassandra community invites you to join us as we take Casssandra Days to China in December!
+
+=== Background
+
+Cassandra Days focus on the open source Apache Cassandra project and the community that supports the project. Everyone (whether they are an individual user, contributor, or company) is welcome to attend and help organize these events.
+
+These events are an opportunity for Apache Cassandra users, enthusiasts, and community members to share their experiences working with Cassandra daily, hear talks and participate in workshops for NoSQL beginners & experts.
+
+=== Event info
+
+In today's world, chip computing power continues to rise. The IoT, AI, and machine vision have multiplied the growth of data. Big data technology has become a bridge between the chip to business, bringing new insights to enterprises, developers, and researchers. All major enterprises across the country are constantly racing on various tracks, with data pouring in from all corners of the country. How do they handle these data, and what supports their smooth experience and seamless connection? 
+
+Cassandra Day China 2022 will unveil the secret for you! 
+
+In order to help more developers understand Cassandra, Cassandra Day China 2022 will be held online on December 22, and will involve technical experts from well-known enterprises including http://www.itcgb.com/[China Golden Bridge], https://www.mgtv.com/[Mango TV], https://www.360.cn/[360], https://www.azul.com/[Azul], https://www.huawei.com[Huawei], https://www.intel.com[Intel Corporation] and https://www.datastax.com[DataStax] to share Cassandra's relevant application experience from the perspective of actual users, help more developers build more convenient, safe and easy to use applications, and discover more possibilities! There will be expert Q&A plus prizes from our sponsors!
+
+=== Agenda
+
+. Market outlook of China's open-source database market - Lu Dongming @ China Golden Bridge
+. Cassandra's practice in Mango Data Marketing Service  - Chen Cong @ Mango TV
+. Multi-datacenter online migration practice - Guohao @ 360
+. Configuration check & JVM performance optimization - Que Naizhen @ Azul
+. Zero Downtime Migration (ZDM) - Wei Deng @ DataStax
+. Contributions to a faster Cassandra - Intel
+. Expert Q&A plus prizes
+
+=== Acknowledgements
+
+Huge thanks to event sponsors https://www.azul.com[Azul Systems] and https://www.intel.com[Intel Corporation], plus support from http://www.itcgb.com[China Golden Bridge] and the https://www.apache.org[Apache Software Foundation].
```

Comment:
Can we add to the page
```
===  Anti-harassment Policy

China Golden Bridge is dedicated to a harassment-free experience for everyone. Our anti-harassment policy can be found [here](https://www.gbconsulting.cn/cdaychina/policy).

"Cassandra Day China ‘22” operates under the terms of the [ASF’s Code of Conduct](https://www.apache.org/foundation/policies/conduct.html).
```

Since the event does not yet have its own page hosted anywhere yet for us to check, and this is one of the hard requirements to get past PMC approval.

Reply:
+1 Added in new commit. 👍